### ISSUE TITLE
NET log directory from config file not detected

### ISSUE THREAD

**Issue by uturunku1** [newrelic/newrelic-diagnostics-cli#23]
[NOTE]: # ( ^^ Provide a general summary of the issue in the title above. ^^ )

## Description

We look at env vars and config files for the agent's log locations.  However, we are failing at pick up the location from the .NET agent's config file. This is how the configuration looks like:

```
<configuration xmlns="urn:newrelic-config" agentEnabled="true">
  <service licenseKey="MY-LICENSE-KEY" />
  <application>
    <name>MY-APP-NAME (Prod)</name>
  </application>
  <log level="info" directory="I:\Logs" />
  <transactionTracer enabled="true" transactionThreshold="apdex_f" stackTraceThreshold="500" recordSql="obfuscated" explainEnabled="false" explainThreshold="500" />
  ```

## Suggested solution

The solution to this problem is to add the path/fields that we must look in a config xml file and add it to the list of `keysInConfigFile`:
https://github.com/newrelic/newrelic-diagnostics-cli/blob/main/tasks/base/log/logHelpers.go#L48

**Comment by uturunku1** [comment]
https://github.com/newrelic/newrelic-diagnostics-cli/pull/55
